### PR TITLE
Lua_api.txt: Add documentation for biome definition

### PR DESCRIPTION
**Pull request by paramat**
First draft.

### CODE REVIEW

**Comment by 0-afflatus** [comment]
I wonder if you should specifically state that heat_point and humidity_point are voronoi points. The word "Characteristic" doesn't really make it clear that these are middle-of-the-range values.

**Comment by paramat** [comment]
Updated, added more explanation.

File: doc/lua_api.txt [hunk 1]
```diff
@@ -3578,6 +3578,51 @@ Definition tables
...
+    --  ^ Ground below air, water below air, ground below water.
+    --  ^ The selected biome then stays in effect for all nodes below until
+    --  ^ column base or the next biome recalculation.
+        heat_point = 0,
+        humidity_point = 50,
+    --  ^ Characteristic average temperature and humidity for the biome.
+    --  ^ These values create 'biome points' on a voronoi diagram that has heat
+    --  ^ and humidity as axes. The resulting voronoi cells determine which
+    --  ^ heat/humidity points belong to which biome, and therefore determine
+    --  ^ the area and location of each biome in the world.
```
Comment (0-afflatus) [inline doc/lua_api.txt]: "The resulting voronoi cells ..." to here.
Does the explanation work without this sentence?
or perhaps
"The resulting voronoi diagram is used to determine the area and location of each biome based on heat and humidity."

Comment (paramat) [inline doc/lua_api.txt]: These docs will probably need improvement later but i'll merge this for now.

Comment (0-afflatus) [inline doc/lua_api.txt]: It does the job :+1:

**Comment by paramat** [comment]
5292ba7391cd611c60ca55df55246af7edefb16c

**Pull request closed without merging**